pset-functions.js

```diff
@@ -208,6 +396,25 @@ console.log('Problem 5:')
 
 // Add your code below this line
 
+const wordTest = "loop"
+
+hasDoubleLetters(wordTest)
+
+function hasDoubleLetters(word) {
+
+ let hasDoubles = false
+
+  for (var i = 0; i < word.length; i++) {
+  //  console.log(word.charAt(i))
+  if (word.charAt(i) === word.charAt(i+1)) {
+    hasDoubles = true
+  }
+
+  }
+  console.log(`${wordTest} has double letters = ${hasDoubles}`)
+
+}
+
```

Comment:
@thetripler Good job here with this solution you have. I see your looping through each letter of the word checking it with the next consecutive letter to verify if they are doubles.
Take a look at this approach [here](https://github.com/jsr20190826/hw-02-functions-solution/blob/a4d789161886ff115657133c279062df8a4ca72d/pset-functions.js#L300)